### PR TITLE
DOCSUP-10063: minimal web UI + new adopter

### PR DESCRIPTION
**Pull request by olgarev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #16158

### CODE REVIEW

File: docs/en/interfaces/http.md [hunk 1]
```diff
@@ -7,16 +7,22 @@ toc_title: HTTP Interface
...
 ``` bash
 $ curl 'http://localhost:8123/'
 Ok.
 ```
 
-Use GET /ping request in health-check scripts. This handler always returns “Ok.” (with a line feed at the end). Available from version 18.12.13.
+Web UI can be accessed here:
+
+``` bash
+$ curl 'http://localhost:8123/play'
```
Comment (alexey-milovidov) [inline docs/en/interfaces/http.md:22]: It is meaningless if you download the HTML source code of Web UI with curl.
A screenshort of a web browser should be attached instead.

File: docs/ru/interfaces/http.md [hunk 2]
```diff
@@ -5,30 +5,36 @@ toc_title: "HTTP-интерфейс"
...
 ``` bash
 $ curl 'http://localhost:8123/'
 Ok.
 ```
 
-В скриптах проверки доступности вы можете использовать GET /ping без параметров. Если сервер доступен всегда возвращается «Ok.» (с переводом строки на конце).
+Веб-интерфейс доступен по адресу:
+
+``` bash
+$ curl 'http://localhost:8123/play'
```
Comment (alexey-milovidov) [inline docs/ru/interfaces/http.md:21]: .

File: docs/ru/interfaces/http.md [hunk 3]
```diff
@@ -5,30 +5,34 @@ toc_title: "HTTP-интерфейс"
...
 $ curl 'http://localhost:8123/'
 Ok.
 ```
 
-В скриптах проверки доступности вы можете использовать GET /ping без параметров. Если сервер доступен всегда возвращается «Ok.» (с переводом строки на конце).
+Веб-интерфейс доступен по адресу: `http://localhost:8123/play`. 
+
+![Веб-интерфейс](../images/play.png#)
+
+В скриптах проверки доступности вы можете использовать `GET /ping` без параметров. Если сервер доступен всегда возвращается «Ok.» (с переводом строки на конце).
```
Comment (adevyatova) [inline docs/ru/interfaces/http.md:22]:
```suggestion
В скриптах проверки доступности вы можете использовать `GET /ping` без параметров. Если сервер доступен, всегда возвращается «Ok.» (с переводом строки на конце).
```

File: docs/ru/interfaces/http.md [hunk 4]
```diff
@@ -75,9 +79,9 @@ ECT 1
 , expected One of: SHOW TABLES, SHOW DATABASES, SELECT, INSERT, CREATE, ATTACH, RENAME, DROP, DETACH, USE, SET, OPTIMIZE., e.what() = DB::Exception
 ```
 
-По умолчанию, данные возвращаются в формате TabSeparated (подробнее смотри раздел «Форматы»).
+По умолчанию данные возвращаются в формате [TabSeparated](formats.md#tabseparated).
 
-Можно попросить любой другой формат - с помощью секции FORMAT запроса.
+Можно попросить любой другой формат с помощью секции FORMAT запроса.
```
Comment (adevyatova) [inline docs/ru/interfaces/http.md:84]:
```suggestion
Можно указать любой другой формат с помощью секции FORMAT запроса.
```

File: docs/ru/interfaces/http.md [hunk 5]
```diff
@@ -5,30 +5,34 @@ toc_title: "HTTP-интерфейс"
...
 Ok.
 ```
 
-Запрос отправляется в виде URL параметра с именем query. Или как тело запроса при использовании метода POST.
+Запрос отправляется в виде URL параметра с именем `query`. Или как тело запроса при использовании метода POST.
 Или начало запроса в URL параметре query, а продолжение POST-ом (зачем это нужно, будет объяснено ниже). Размер URL ограничен 16KB, это следует учитывать при отправке больших запросов.
 
-В случае успеха, вам вернётся код ответа 200 и результат обработки запроса в теле ответа.
-В случае ошибки, вам вернётся код ответа 500 и текст с описанием ошибки в теле ответа.
+В случае успеха вам вернётся код ответа 200 и результат обработки запроса в теле ответа.
```
Comment (adevyatova) [inline docs/ru/interfaces/http.md:32]:
```suggestion
В случае успеха возвращается код ответа 200 и результат обработки запроса в теле ответа, в случае ошибки — код ответа 500 и текст с описанием ошибки в теле ответа.
```

File: docs/ru/interfaces/http.md [hunk 6]
```diff
@@ -5,30 +5,34 @@ toc_title: "HTTP-интерфейс"
...
+В случае ошибки вам вернётся код ответа 500 и текст с описанием ошибки в теле ответа.
```
Comment (adevyatova) [inline docs/ru/interfaces/http.md:33]:
```suggestion
```

**Comment by olgarev** [comment]
@alexey-milovidov Hi! I've added the screenshots, and 'doc check' have passed correctly. But now we have errors in our local building process. Can smb help us with that? When this PR is merged, we all will have problems (
image

File: docs/en/interfaces/http.md (hunk 1 above)
Comment (olgarev) [reply docs/en/interfaces/http.md]: @alexey-milovidov Fixed, thanks

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
Ok.

PS. Sometimes I want to emphasize that this is just a *minimal* web UI.

**Pull request merged**